Makefile

```diff
@@ -52,18 +52,16 @@ mypy: ## run mypy checks
 	MYPYPATH=$(CWD)/mypy-stubs mypy parsl/app/ parsl/channels/ parsl/dataflow/ parsl/data_provider/ parsl/launchers parsl/providers/
 
 .PHONY: local_thread_test
-local_thread_test: $(DEPS) ## run all tests with local_thread config
+local_thread_test: ## run all tests with local_thread config
```

Comment:
Removing DEPS here would result in tests possibly running without the test-requirements installed, no?

Reply:
Resolved over chat. We are decoupling the tests from building to avoid having to rebuild everytime for every small test.